src/actions/openModalHandler.ts

```diff
@@ -0,0 +1,8 @@
+import { IOpenModal } from "../types/types";
+
+
```

Comment:
Delete this blank space, please.

Reply:
Solved c049ae1d0d5fda6f5c5d683bfcbf713caa123bb3.